README.Md

```diff
@@ -26,9 +26,9 @@ might be supported in a near future.
 
 Getting Started
 ===============
-There is much more information, including a more complete (["Getting Started Guide"](https://www.atomvm.net/doc/master/getting-started-guide.html)), ([extensive documentation](https://www.atomvm.net/doc/master/build-instructions.html)), ([examples](https://www.atomvm.net/sample-code)), detailed ([build instructions](https://www.atomvm.net/doc/master/build-instructions.html)) and ([contact information](https://www.atomvm.net/contact)) available on the [AtomVM](https://atomvm.net) project website.
+There is much more information, including a more complete ["Getting Started Guide"](https://www.atomvm.net/doc/master/getting-started-guide.html), [extensive documentation](https://www.atomvm.net/doc/master/build-instructions.html), [examples](https://www.atomvm.net/sample-code), detailed [build instructions](https://www.atomvm.net/doc/master/build-instructions.html) and [contact information](https://www.atomvm.net/contact) available on the [AtomVM](https://atomvm.net) project website.
```

Comment:
Since we are changing this block of text, would you mind trying to fit as much as possible into 100 character columns?
Let's change it while changing the text.